chat-server/src/services/database.ts

```diff
@@ -2,37 +2,37 @@ import { Db, MongoClient, ObjectId } from 'mongodb';
 import { initiateLogger } from './logger';
 
 // TODO: define interfaces for database documents
-// interface StaticAsset {
-//   checksum: string;
-//   key: string;
-// }
-
-// export interface AssetDocument {
-//   _id: string;
-//   data: BinaryData;
-// }
-
-// interface PageDocument {
-//   _id: ObjectId;
-//   page_id: string;
-//   filename: string;
-//   ast: object;
-//   source: string;
-//   static_assets: StaticAsset[];
-//   build_id: ObjectId;
-//   created_at: Date;
-// }
-
-// interface UpdatedPageDocument {
-//   _id: ObjectId;
-//   page_id: string;
-//   filename: string;
-//   ast: object;
-//   static_assets: StaticAsset[];
-//   created_at: Date;
-//   updated_at: Date;
-//   deleted: boolean;
-// }
+interface StaticAsset {
+  checksum: string;
+  key: string;
+}
+
+export interface AssetDocument {
+  _id: string;
+  data: BinaryData;
+}
+
+interface PageDocument {
+  _id: ObjectId;
+  page_id: string;
+  filename: string;
+  ast: object;
```

Comment:
Why is this needed?

We'll probably end up merging these definitions soon anyway, right?

Btw, TSC should complain about using `object` as such...

Reply:
this was just to get tsc to compile. 

will be deleted once we start populating with our own code.

and re usage of `object`, pls feel free to disallow that when you set up the tsconfig.